libqtile/widget/graph.py

```diff
@@ -237,12 +244,19 @@ def update_graph(self):
 
 
 def get_meminfo():
-    with open('/proc/meminfo') as file:
-        val = {}
+    val = {}
+    proc = '/proc/meminfo'
+    if platform.system() == "FreeBSD":
+        proc = "/compat/linux" + proc
+    with open(proc) as file:
         for line in file:
-            key, tail = line.split(':')
-            uv = tail.split()
-            val[key] = int(uv[0])
+            if line.lstrip().startswith("total"):
+                pass
+            else:
+                key, tail = line.strip().split(':')
+                uv = tail.split()
+                val[key] = int(uv[0]) // 1000
```

Comment:
It looks like you added `// 1000`, is that a BSD specific, or is that also fixing some error?